10-Modelling-Deformable-Interfaces-with-Meshes/02-Vesicle-MD.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>I'm not sure what this code block is trying to convey. The tether parameter looks to belong with the "mesh_bond_potential" code below. The rest of the code looks like it belongs in its own block with some explanation of how these values might be used to validate the tether parameter choices?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>